tracer/test/snapshots/StackExchangeRedisTests.V1_0_414.SchemaV0.verified.txt

```diff
@@ -8,15 +8,16 @@
     Type: redis,
     Tags: {
       component: StackExchangeRedis,
-      db.redis.database_index: 1,
       env: integration_tests,
       out.host: stackexchangeredis,
       out.port: 6379,
       redis.raw_command: APPEND ssetkey,
       runtime-id: Guid_1,
-      span.kind: client
+      span.kind: client,
+      _dd.p.dm: -0
     },
     Metrics: {
+      db.redis.database_index: 1.0,
```

Comment:
Wasn't that part of Andrew's PR already? Why does it change now only? It's because it's part of comprehensive tests and wasn't updated before?

Reply:
It turns out this file doesn't run in our automated tests (not even the comprehensive testing!) but I wanted to make sure all the possible snapshots were updated. I should look into including either 1.0 or 1.x packages in our CI for better compatibility

Reply:
It looks like this is the same issue as the SQS tests: These early versions can only run on .NET Framework, but since this relies on a linux container, we can't test this in CI at the moment.